.ci.yaml

```diff
@@ -2594,6 +2594,7 @@ targets:
 
   - name: Mac dart_plugin_registry_test
     recipe: devicelab/devicelab_drone
+    bringup: # Flaky https://github.com/flutter/flutter/issues/99940
```

Comment:
are we missing `True` before the comment?

Reply:
Yes! Added.